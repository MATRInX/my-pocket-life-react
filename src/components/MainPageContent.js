import React from 'react';
import MainBody from './MainBody';
import MainSidebar from './MainSidebar';
import LastPostsCarousel from './LastPostsCarousel';


const MainPageContent = () => (
  <div>
  MainPageContent...
    <LastPostsCarousel />
    <MainBody />
    <MainSidebar />
  </div>
);

export default MainPageContent;