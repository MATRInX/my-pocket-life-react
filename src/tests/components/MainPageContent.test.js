import React from 'react';
import { shallow } from 'enzyme';
import MainPageContent from '../../components/MainPageContent';

test('should correctly render MainPageContent', () => {
  const wrapper = shallow(<MainPageContent />);
  expect(wrapper).toMatchSnapshot();
});