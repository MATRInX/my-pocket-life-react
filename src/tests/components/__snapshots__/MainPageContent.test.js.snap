// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render MainPageContent 1`] = `
<div>
  MainPageContent...
  <LastPostsCarousel />
  <MainBody />
  <MainSidebar />
</div>
`;
